rspace/src/main/scala/coop/rchain/rspace/history/HistoryRepository.scala

```diff
@@ -45,10 +43,10 @@ trait HistoryRepository[F[_], C, P, A, K] extends ChannelStore[F, C] {
 
   def importer: F[RSpaceImporter[F]]
 
-  def stateMerger: StateMerger[F]
-
   def getHistoryReader(stateHash: Blake2b256Hash): HistoryReader[F, Blake2b256Hash, C, P, A, K]
 
+  def getSerializeC: Serialize[C]
```

Comment:
I have to add this to be able to create index for joins. If we go towards storing in joins just hashes we might be able to remove it.